src/js/utils/option-utils.jsx

```diff
@@ -183,7 +183,7 @@ export const debounceProductsInOrders = (waitTime, minSearchLength, vendor, dest
 export const debounceOrganizationsFetch = (waitTime, minSearchLength) =>
   _.debounce((searchTerm, callback) => {
     if (searchTerm && searchTerm.length >= minSearchLength) {
-      apiClient.get(`/openboxes/api/organizations?q=${searchTerm}`)
+      apiClient.get(`/openboxes/api/organizations/listSuppliers?q=${searchTerm}`)
```

Comment:
Therefore, this should be 
```
apiClient.get(`/openboxes/api/organizations?q=${searchTerm}&roleType=ROLE_SUPPLIER`)
```
Since there's only one value here, you might need to add some processing in the list controller action to convert the parameter to a list.
```
params.roleType = params.list("roleType")
```
In addition you might need to also convert it from a string to the enum.
```
params.roleType = params.list("roleType").collect { return it as RoleType }
```

Reply:
That's right. I didn't even have to add any processing since it was already added. I've tried to do it that way at the beggining but instead of "roleType" parameter I added "roleTypes" so I had a typo.